Homebrew install of Kong fails on MacOS

==> Installing kong from kong/kong
==> Installing dependencies for kong/kong/kong: coreutils, kong/kong/openresty@1.13.6.2 and libyaml
==> Installing kong/kong/kong dependency: coreutils
==> Downloading https://homebrew.bintray.com/bottles/coreutils-8.31.mojave.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/f1/f11898e59abf1c9ea9807ab15c7cdfc930bbfe1df14e432e5d2a89f11c405445?__gda__=exp=1564681613~hmac=4c16ecfc11
######################################################################## 100.0%
==> Pouring coreutils-8.31.mojave.bottle.tar.gz
==> Caveats
Commands also provided by macOS have been installed with the prefix "g".
If you need to use these commands with their normal names, you
can add a "gnubin" directory to your PATH from your bashrc like:
  PATH="/usr/local/opt/coreutils/libexec/gnubin:$PATH"
==> Summary
🍺  /usr/local/Cellar/coreutils/8.31: 476 files, 8.8MB
==> Installing kong/kong/kong dependency: kong/kong/openresty@1.13.6.2
==> Downloading https://raw.githubusercontent.com/Kong/openresty-build-tools/0.0.5/kong-ngx-build
######################################################################## 100.0%
==> ./kong-ngx-build --prefix /usr/local/Cellar/openresty@1.13.6.2/1.13.6.2 --openresty 1.13.6.2 --openssl 1.1.1c --luarocks 3.1.3 -j 12
Last 15 lines from /Users/lakshman/Library/Logs/Homebrew/openresty@1.13.6.2/01.kong-ngx-build:
 + ngx_stream_lua_module was configured
adding module in /private/tmp/openresty@1.13.6.2-20190801-6150-1d186t9/work/lua-kong-nginx-module
 + ngx_http_lua_kong_module was configured
checking for PCRE library ... not found
checking for PCRE library in /usr/local/ ... not found
checking for PCRE library in /usr/include/pcre/ ... not found
checking for PCRE library in /usr/pkg/ ... not found
checking for PCRE library in /opt/local/ ... not found

./configure: error: the HTTP rewrite module requires the PCRE library.
You can either disable the module by using --without-http_rewrite_module
option, or install the PCRE library into the system, or build the PCRE library
statically from the source with nginx by using --with-pcre=<path> option.

ERROR: failed to run command: sh ./configure --prefix=/usr/local/Cellar/openresty@1.13.6.2/1.13.6.2/openresty/nginx \...

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,5 +1,6 @@
 language: objective-c
 os: osx
+osx_image: xcode11
 
 before_install:
   - brew update >/dev/null

--- a/Formula/openresty.rb
+++ b/Formula/openresty.rb
@@ -17,6 +17,7 @@ class Openresty < Formula
 
   option "with-debug", "Compile with support for debug logging but without proper gdb debugging symbols"
 
+  depends_on "pcre"
   depends_on "coreutils"
   conflicts_with "kong/kong/luarocks", :because => "We switched over to a new build method and LuaRocks no longer needs to be installed separately. Please remove it with \"brew remove kong/kong/luarocks\"."
 

--- a/Formula/openresty@1.13.6.2.rb
+++ b/Formula/openresty@1.13.6.2.rb
@@ -17,6 +17,7 @@ class OpenrestyAT11362 < Formula
 
   option "with-debug", "Compile with support for debug logging but without proper gdb debugging symbols"
 
+  depends_on "pcre"
   depends_on "coreutils"
   conflicts_with "kong/kong/luarocks", :because => "We switched over to a new build method and LuaRocks no longer needs to be installed separately. Please remove it with \"brew remove kong/kong/luarocks\"."
 

